README.md

```diff
@@ -42,6 +42,27 @@ $ IMG=quay.io/user/internal-services:my-tag make docker-push
 To test the code, simply run `make test`. This command will fetch all the required dependencies and test the code. The
 test coverage will be reported at the end, once all the tests have been executed.
 
+### Accessing the logs
+
+#### CloudWatch
+
+The procedure to request access to CloudWatch is documented in the following link: [Getting Access](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/stonesoup/sop/getting-access.md#gain-view-access-to-appsre-logs-including-logs-from-the-fleet-manager-cluster-and-argocd).
+Make sure MFA is enabled to your account, otherwise it will not be possible to access the logs.
+
+Once in the CloudWatch web console you can search for `stonesoup-int-svc` in the log group search box.
+
+#### Splunk
+
+First access the [Splunk & app-sre](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/splunk.md#getting-support-working-with-it) page and complete the "Access to Monitoring Platform" request.
+
```

Comment:
Probably not necessary to specify this here.

Reply:
I think we need as it is not easy to find this link to get the access to Splunk. I understood from @scoheb that Internal Service logs should also be available in Splunk. Maybe I should rewrite this one as well?

Reply:
Weird. I don't remember commenting this and the comment itself doesn't make much change... Anyway, resolving.